quickstarts/aws/aws-ec2/dashboards/aws-ec2.json

```diff
@@ -0,0 +1,199 @@
+{
+	"name": "AWS EC2",
+	"description": "",
+	"permissions": "PUBLIC_READ_WRITE",
```

Comment:
```suggestion
```

Reply:
This will actually not work. In the future you can run this sanitize dashboard script found here to sanitize any dashboard JSON that is copied from NR1 and remove this field. https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboard-json

Reply:
Forgot that! Thanks for the script.